extern/NEZplug++/divfix.h

```diff
@@ -0,0 +1,18 @@
+

```

Comment:
OK, so here is a real question.
Why NEZplug++? I thought this was only for NSFplay...